### ISSUE TITLE
CoxTimeVaryingFitter(): Getting wrong estimates in 

### ISSUE THREAD

**Issue by lakshmiraut** [CamDavidsonPilon/lifelines#961]
My own java program and R both give same parameter estimate but lifelines CoxTimeVaryingFitter() give much different estimates.  Could it be due to normalization used in CoxTimeVaryingFitter() or something else?

**Comment by CamDavidsonPilon** [comment]
Hm, it shouldn't be the normalization (that only should affect the baseline). Can you send me your dataset along with your lifelines & R code? I'd like to reproduce locally as well.

**Comment by lakshmiraut** [comment]
Cameron,
I investigated further and it is my mistake.  My (java code, SAS and) R  using Robust method.  When I took the "robust=TRUE" from my R code, it gave the same results in both R and lifelines in python. But your package has not robust option implemented.  I think it will be very helpful to have it implemented.

It will NICE to have an option to calculate the  cumulative baseline hazard with the option of no normalization.  This will be useful to me and I am sure to others to calculate the cumulative baseline hazard of a newdata which needs to be normalized with mean that one needs to store.  

Thanks.   

Lakshmi
